org.neocities.albertt.flyby.yaml

```diff
@@ -0,0 +1,39 @@
+id: org.neocities.albertt.flyby
+runtime: org.gnome.Platform
+runtime-version: '49'
+sdk: org.gnome.Sdk
+command: org.neocities.albertt.flyby
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --share=network
+modules:
+  - name: gsf
+    config-opts:
+    - "--disable-static"
+    cleanup:
+    - "/include"
+    - "/lib/pkgconfig"
+    - "/share/man"
+    - "/share/gtk-doc/html/gsf"
+    - "/share/thumbnailers"
+    - "/bin/gsf*"
+    - "*.la"
```

Comment:
except these 3

```
- "/share/gtk-doc/html/gsf"
- "/share/thumbnailers"
- "/bin/gsf*"
```

you should move the rest to a toplevel cleanup just before `modules:`.